extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -857,11 +854,15 @@ export class AmpA4A extends AMP.BaseElement {
       return true;
     }
 
-    removeChildren(this.element);
+    // Remove rendering frame, if it exists.
+    if (this.iframe) {
+      this.element.removeChild(this.iframe);
```

Comment:
We're assuming that the frame is nested directly beneath the element which is likely fine.  Other option would be to do this.iframe.parentElement.removeChild(this.iframe).  At the very least, can you wrap this with dev().assert(...) as it should return the element that was removed